docs/source/cli/graph-construction/single-machine-gconstruct.rst

```diff
@@ -147,6 +147,15 @@ GraphStorm provides a set of transformation operations for different types of fe
                   "max_bound": 2.,
                   "min_bound": -2.}
 
+* **Numerical standard transformation** normalizes numerical input features with `val = val / sum`, where `val` is the feature value and `sum` is a summation of all the values in the feature. The ``name`` field in the feature transformation dictionary is ``standard``. The dictionary can contain one optional field, i.e., ``sum``, which is summation of all the values in the feature col from the previous transformation.
```

Comment:
Why is the sum optional here? Why not always determined by the summation value of the data input.

Reply:
The distribution will change between training graph and inference graph if we do not save the summation value.

Reply:
We do save the original value.
The implementation is compatible with GSProcessing.

Reply:
Updated the doc.